ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/GamePanel.java

```diff
@@ -76,13 +81,39 @@ public GamePanel(@NotNull GlassPane glassPane, @NotNull Game game) {
     }
 
     private void initComponents() {
-        setLayout(new GridBagLayout());
+        setLayout(new BoxLayout(this, BoxLayout.LINE_AXIS));
+        Board[] boards = mGame.getBoards();
+
 
         JPanel boardPanels = new JPanel();
         boardPanels.setOpaque(false);
-        boardPanels.setLayout(new GridBagLayout());
+        boardPanels.setLayout(new BoxLayout(boardPanels, BoxLayout.LINE_AXIS));
+        boardPanels.addComponentListener(new ComponentListener() {
+            @Override
+            public void componentResized(ComponentEvent e) {
+                IntStream.range(0, boards.length).forEach(boardIndex -> {
```

Comment:
Is there a reason you didn't use the one I suggested?
`Streams.of(mGameBoards).forEach(board -> {...});`